package.json

```diff
@@ -48,6 +48,7 @@
     "jest-environment-jsdom": "^29.7.0",
     "jscodeshift": "^0.15.2",
     "lint-staged": "^13.0.3",
+    "mime-types": "^2.1.35",
```

Comment:
Why did this need to be added as a direct dependency? It was already in the package-lock file, and we don't import it anywhere in the source code.

Reply:
for some reason the test suite was erroring out because of that dependency missing - I tried to clean up the codebase and reproduce but I can't, so I'll revert that commit.